src/keys/mod.rs

```diff
@@ -5,8 +5,8 @@
 #[cfg_attr(docsrs, doc(cfg(feature = "ternary_keys")))]
 pub mod ternary;
 
-#[cfg(feature = "pbkdf")]
-#[cfg_attr(docsrs, doc(cfg(feature = "pbkdf")))]
+#[cfg(feature = "pbkdf2")]
+#[cfg_attr(docsrs, doc(cfg(feature = "pbkdf2")))]
 pub mod pbkdf;
```

Comment:
Should the module itself be pbkdf2 in case we add more pbkdf ?

Reply:
Ah I see that inside you have features for pbkdf2, ignore me

Reply:
PBKDF2 is a concrete algorithm, pbkdf is password-based key derivation function in general. Scrypt, for example, will nicely fit pbkdf, but not pbkdf2. If there are other pbkdfs we will declare this mod via `cfg(any(feature = "pbkdf2", feature = "scrypt"))`